plugins/ofl/export.js

```diff
@@ -0,0 +1,18 @@
+module.exports.name = 'Open Fixture Library JSON';
```

Comment:
As you wish. I think your initiative has the potential to make a future standard, as UFO font file. Unified Font Object. and be opened and saved by different vendors.